members/I000024.yaml

```diff
@@ -1,87 +1,13 @@
 bioguide: I000024
 contact_form:
   method: POST
-  action: http://www.inhofe.senate.gov/contact
+  action: ""
   steps:
-    - visit: "http://www.inhofe.senate.gov/contact"
+    - visit: "https://www.inhofe.senate.gov/contact/email-jim"
     - select:
-      - name: subject
-        selector: "#subject"
-        value: "$TOPIC"
-        required: true
-        options:
-          - "Adoption and Abortion"
-          - "Agriculture and Trade"
-          - "Education"
-          - "Energy and Environment"
-          - "Federal Spending"
-          - "Healthcare"
-          - "Immigration"
-          - "National Defense / Homeland Security"
-          - "Second Amendment"
-          - "Social Security"
-          - "Taxation"
-          - "Traditional Values"
-          - "Transportation"
-          - "Veterans"
-          - "Other"
-    - find:
-      - selector: "#continue-form"
-    - fill_in:
-      - name: fname
-        selector: "#fname"
-        value: $NAME_FIRST
-        required: true
-      - name: lname
-        selector: "#lname"
-        value: $NAME_LAST
-        required: true
-      - name: mailing_streetAddress1
-        selector: "#mailing_streetAddress1"
-        value: $ADDRESS_STREET
-        required: true
-      - name: mailing_streetAddress2
-        selector: "#mailing_streetAddress2"
-        value: $ADDRESS_STREET_2
-        required: false
-      - name: mailing_city
-        selector: "#mailing_city"
-        value: $ADDRESS_CITY
-        required: true
-      - name: mailing_zipCode
-        selector: "#mailing_zipCode"
-        value: $ADDRESS_ZIP5
-        required: true
-      - name: home_phone_number
-        selector: "#home_phone_number"
-        value: $PHONE
-        required: false
-      - name: email_address
-        selector: "#email"
-        value: $EMAIL
-        required: true
-      - name: verify_email
-        selector: "#verify_email"
-        value: $EMAIL
-        required: true
-      - name: message
-        selector: "#message"
-        value: $MESSAGE
-        required: true
-    - javascript:
-      - name: Message
-        selectors: [ "#message" ]
-        values: [ "$MESSAGE" ]
-        commands: [ "elements[0].value = elements[0].value.replace(/https:\\/\\/www\\.usagainstalzheimers\\.org\\/sites\\/default\\/files\\/\S+\\.pdf/gi, 'usagainstalzheimers.org').replace(/https:\\/\\/www\\.alz\\.org\\/media\\/HomeOffice\\/\S+\\.pdf/gi, 'alz.org').replace(/[“]/g,'')"]
-        required: true
-    - check:
-      - name: response
-        selector: "#response_y"
-        value: "Y"
-    - select:
-      - name: Your Prefix
-        selector: "#salutation"
-        value: "$NAME_PREFIX"
+      - name: Prefix
+        selector: select#input-B03E0527-4040-F985-52CD-C0DD6BEF325F
```

Comment:
Are these generated frequently or relatively stable?

Reply:
Hard to say. My read is that they can be considered relatively stable, but can't say for sure. I tried looking for other ways to select these elements on the page but didn't see anything more unique or more stable.